### PR TITLE
Address review feedback for form end guide

### PR DESCRIPTION
**Pull request by lognaturel**
Thanks for the great feedback, @seadowg and @alyblenkin!

I believe I have addressed all other than:
- did not add image re-enforcing that send and finalize are the same action. I couldn't quickly figure out how to make it flow and I'm not sure we want to draw attention to that distinction, necessarily. Will think about it more.
- did not add a nudge to split the workflow into multiple forms. I couldn't find a great place to put it.

### CODE REVIEW

File: docs/form-audit-log.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ For the most accurate locations, set ``location-priority`` to `high-accuracy`. F
 
   Disabling location tracking will not prevent users from filling out forms, but these changes are logged as events in the log.
 
-.. _viewing-audit-logs:
+.. _form-audit-log-change-tracking:
```
Comment (yanokwa) [inline docs/form-audit-log.rst:82]: Not consistent with previous tag of `_audit-geolocation-tracking`. Maybe change that or change these ones you've added?

File: docs/guide-end-of-form.rst [hunk 2]
```diff
@@ -40,6 +40,7 @@ Finalized forms are ready to be sent and can't be edited. The ``finalized`` stat
 
 * give data collectors control over when filled forms are queued for submission.
 * reduce the risk that data collectors continue editing data after they no longer have access to the data collection subject.
+* prevent edits after an important workflow step is completed (for example, supervisor review)
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:43]: Add period

File: docs/guide-end-of-form.rst [hunk 3]
```diff
@@ -72,12 +73,12 @@ Auto send setting
 
 We generally recommend turning on :guilabel:`auto send` in :ref:`form management settings <form-management-settings>`. When :guilabel:`auto send` is on, Collect attempts to send filled forms as soon as they are finalized. The benefits of :guilabel:`auto send` are:
 
-* reduced risk of data collectors forgetting to submit data in a timely way.
-* automatically retry failed submissions. On poor or intermittent data connections, this can be very helpful.
+* reduced risk of data collectors forgetting to send data in a timely way.
+* more opportunities to retry sending. On poor or intermittent data connections, this can be very helpful.
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:77]: "Very helpful on poor or intermittent Internet connections" reads better to me. Data connections sounds weird for a data collection app.

File: docs/guide-end-of-form.rst [hunk 4]
```diff
@@ -114,38 +115,55 @@ You can also use :ref:`draft names <instance-name>` to include information about
 
 If you use a constraint as in the above example, the user will need to come back into the draft and change their answer to the question before they can send the data.
 
-If your users have many drafts and will only need to edit a few before sending them, you can use a ``note`` without a constraint to guide users to save as draft. They will then be able to use the :ref:`bulk finalization <bulk-finalizing-drafts>` functionality when they are ready to submit.
+If your users have many drafts, will only need to edit a few before sending them, but don't know which ones will need to be edited, you can use a ``note`` without a constraint to guide users to save as draft.
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:118]: You are missing `and` after the first comma.

File: docs/guide-end-of-form.rst [hunk 5]
```diff
@@ -114,38 +115,55 @@ You can also use :ref:`draft names <instance-name>` to include information about
...
+To eventually send, someone can either show the button, or use the :guilabel:`Finalize all drafts` functionality from the :guilabel:`Drafts` list.
+
 Remove save draft (:fa:`floppy-disk`) while filling a form
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Collect's :ref:`protected access control settings <admin-settings>` also contain a :guilabel:`Form Entry Settings` section for hiding actions available from the form filling screen. You can hide the Save (:fa:`floppy-disk`) button from the menu and from the :ref:`form exit dialog <exit-form-filling>`. This will prevent data collectors from saving as draft during a form filling session.
 
 You may still want to allow them to save as draft from the form end screen if, for example, it's appropriate for them to make small edits after all of the initial data is captured.
 
+.. _guide-form-states-hide-save-draft-form-end:
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:144]: A lot of your anchors use `guide-form-states`. Maybe `guide-end-form...`

File: docs/guide-end-of-form.rst [hunk 6]
```diff
@@ -160,23 +178,25 @@ In many cases, the admin password will never need to be used: its purpose is onl
...
+Some questions to ask yourself and your data collectors as you design your workflow are:
 
 * Is it possible for a data collector to reach the end of the form but still have information to fill in?
 * What should happen if a data collector is interrupted while filling out a form?
 * How capable are data collectors of remembering a workflow?
 * How many times will data collectors repeat the same workflow?
 * How capable are data collectors of making independent decisions when faced with unexpected situations like an interview being interrupted?
 * How trusted and well-trained are data collectors? Are they likely to want to "cheat" in some way to save time and/or effort?
-* What are the consequences of incorrect data being submitted? What are the next steps if that happens and is detected?
+* Will data collectors be in a distracting or stressful environment that could reduce their usual capacities or honesty?
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:189]: s/honesty/performance

File: docs/guide-end-of-form.rst [hunk 7]
```diff
@@ -185,23 +205,41 @@ In many workflows, it's important to guarantee that data is not changed after th
 
 When data collectors reach the form end screen, they only have the option to :guilabel:`Finalize`. If they are interrupted during a form filling session, they need to exit and discard changes or rely on automatic data backups and recovery (the partially-filled form will open automatically when they open the same blank form again).
 
-Edits may be needed to some drafts
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In some workflows, new information may need to be added to a form after a first data collection event. For example:
+.. note::
+  If you would still like to leave the door open to edits but they are generally strongly discouraged, you could leave the option to save as draft from the form filling experience and use :ref:`audit logging <guide-form-states-form-audit-log>` with change tracking and reasons for change to get more information on edits that are made.
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:209]: s/leave the door open to/allow for/

You can link directly to change tracking now.

File: docs/guide-end-of-form.rst [hunk 8]
```diff
@@ -185,23 +205,41 @@ In many workflows, it's important to guarantee that data is not changed after th
...
-* review to fix small mistakes like typos may be encouraged.
 * tasks like transcribing an audio recording may be needed.
 
+This can generally be addressed in form design by using required questions. You can also use a ``note``, a ``hint``, or a ``required_message`` to prompt users to exit the form and :guilabel:`Save as draft` at known points of the form.
+
+If you have designed your form so that it's only possible to reach the form end screen once all necessary tasks are complete, you may want to :ref:`hide the Save as draft button from the form end screen <guide-form-states-hide-save-draft-form-end>`. It can be easier for data collectors to take the right action if they always use the back button to save as draft and only use the form end screen to finalize/send.
+
+Edits are needed but it's not known ahead of time to which drafts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In other workflows, edits are needed but it is not known ahead of time which drafts will be edited. For example:
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:225]: "In other workflows, it is not known ahead of time which drafts will be edited." So it's not the same as heading

**Review by yanokwa — CHANGES_REQUESTED** [review]

File: docs/guide-end-of-form.rst [hunk 9]
```diff
@@ -185,23 +205,41 @@ In many workflows, it's important to guarantee that data is not changed after th
...
 * a form may capture data from multiple days.
-* a data collection subject may be observable at different times or from different angles, revealing new information.
-* review to fix small mistakes like typos may be encouraged.
 * tasks like transcribing an audio recording may be needed.
 
+This can generally be addressed in form design by using required questions. You can also use a ``note``, a ``hint``, or a ``required_message`` to prompt users to exit the form and :guilabel:`Save as draft` at known points of the form.
+
+If you have designed your form so that it's only possible to reach the form end screen once all necessary tasks are complete, you may want to :ref:`hide the Save as draft button from the form end screen <guide-form-states-hide-save-draft-form-end>`. It can be easier for data collectors to take the right action if they always use the back button to save as draft and only use the form end screen to finalize/send.
+
+Edits are needed but it's not known ahead of time to which drafts
```
Comment (yanokwa) [inline docs/guide-end-of-form.rst:223]: "Edits are needed but it's uncertain which ones in advance"

File: docs/guide-end-of-form.rst (hunk 3 above)
Comment (lognaturel) [reply docs/guide-end-of-form.rst:77]: Pretty sure you wrote that one...

File: docs/guide-end-of-form.rst (hunk 4 above)
Comment (lognaturel) [reply docs/guide-end-of-form.rst:118]: It's a list with three items. Maybe changing the but to and is clearer?

**Review by yanokwa — APPROVED** [review]

**Pull request merged**